server/publicapi/features/prepopulate.feature

```diff
@@ -7,7 +7,7 @@ Feature: Prepopulate public content
     	"""
```

Comment:
I agree. The thing is realized that we need to improve our behave testing data set if we want to write good tests, and felt it is kind of beyond the scope of this ticket. Basically the `populate` would need to be changed so that it initializes database with data containing more variation in it.